lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -785,16 +784,16 @@ def ng_update_session_flows(self, request: SessionSet,
         fut.set_result(response)
 
     def _ng_tunnel_update(self, pdr_entry: PDRRuleEntry, subscriber_id: str) -> bool:
-        if pdr_entry.pdr_state == PdrState.Value('INSTALL'):
-            ret = self._classifier_app.add_tunnel_flows(\
-                           pdr_entry.precedence, pdr_entry.local_f_teid,\
-                           pdr_entry.far_action.o_teid, pdr_entry.ue_ip_addr,\
-                           pdr_entry.far_action.gnb_ip_addr, encode_imsi(subscriber_id))
-
-        elif pdr_entry.pdr_state in \
-             [PdrState.Value('REMOVE'), PdrState.Value('IDLE')]:
-            ret = self._classifier_app.delete_tunnel_flows(\
-                           pdr_entry.local_f_teid, pdr_entry.ue_ip_addr)
+        ret = True
```

Comment:
rm?

Reply:
Yes. now we defined new generic method "gtp_handler" and passing argument pdr_state with other parameters.
According to pdr state value, we will call the delete/Add/Idle/Resume/Suspended gtp tunnel.